mwdb/web/src/commons/ui/ObjectLink.js

```diff
@@ -3,14 +3,14 @@ import { Link, useParams } from "react-router-dom";
 
 import { AuthContext } from "../auth";
 import { mapObjectType } from "../helpers";
+import { useRemotePath } from "../remotes/index";
```

Comment:
`index.js` is like `__init__.py`, there is no need to import directly from it.

```suggestion
import { useRemotePath } from "../remotes";
```

Reply:
fixed